### ISSUE TITLE
Issue in pasting some contents containing text and image from CKEditor which was copied from a word file.

### ISSUE THREAD

**Issue by trupti26k** [ckeditor/ckeditor4#5339]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Add some text and image in the word file.
2. Copy contents of the word file and paste it into CKEditor.
3. Please find the file attached.
[test.docx](https://github.com/ckeditor/ckeditor4/files/9681192/test.docx)

### Expected result

*What is the expected result of the above steps?*
Image in the copied content should paste correctly.
Actual word file image is attched below.

![image](https://user-images.githubusercontent.com/29269067/193201458-08496b39-a7a0-47d1-a9f3-6e76930dc402.png)

### Actual result

*What is the actual result of the above steps?*

![image](https://user-images.githubusercontent.com/29269067/193201541-1fc61768-69b3-4f3c-a78a-b9194fc9eb87.png)

## Other details

* Browser: …
* OS: …
* CKEditor version: …
* Installed CKEditor plugins: …

**Comment by Comandeer** [comment]
I can confirm the issue. The pasting results in the "[pastetools-failed-image-extraction](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_errors.html#pastetools-failed-image-extraction)" error. Our parser finds two images in the RTF but only one image in the HTML. This is connected with the fact that the image in the `.docx` file is a composition of two shapes: a rectangle and an ellipse. Additionally, these two images seem to be in unsupported formats (WML and EMF).

To sum up, there are two distinctive issues with the attached `.docx` file:

* lack of support for WML and EMF image formats → #4290,
* lack of support for images composed from several other images/shapes.

As the support for WML and EMF formats is already tracked in a separate issue, I'll change this one to a feature request for the support of composed images.

**Comment by jacekbogdanski** [comment]
It looks like shapes also impact how other images are converted. If a document (precisely, clipboard HTML) includes images mixed with shapes, the code responsible for matching base64 with `img[src]` will fail:
https://github.com/ckeditor/ckeditor4/blob/b520aa2b406091dd378f77b5173a6249a2ddef38/plugins/pastetools/filter/image.js#L208-L215

At the moment I don't see any other workaround for that issue than just implementing shapes, so let's treat it as a part of a feature request.
